activesupport/lib/active_support/test_case.rb

```diff
@@ -84,6 +84,7 @@ def parallelize(workers: :number_of_processors, with: :processes, threshold: Act
         return if workers <= 1
 
         Minitest.parallel_executor = ActiveSupport::Testing::ParallelizeExecutor.new(size: workers, with: with, threshold: threshold)
+        ActiveSupport.test_order = :parallel
```

Comment:
This enables all `ActiveSupport::TestCase` to be runned in parallel by default when using parallel testing.